docs/changelogs/6.2.0_Changelog.rst

```diff
@@ -104,9 +104,8 @@ Removed Modules
 
 pupmod-simp-mcollective and pupmod-simp-activemq
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-* Puppetlabs support for MCollective has been dropped, rendering the SIMP
-  modules to support this, ``pupmod-simp-mcollective`` and
-  ``pupmod-simp-activemq``, non-functional.
+* Puppetlabs no longer supports ``pupmod-simp-mcollective`` and
```

Comment:
This change does not make sense.  Puppetlabs would not be supporting SIMP modules.